pkg/model/segment.go

```diff
@@ -8,6 +8,7 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/bluesky-social/indigo/lex/util"
```

Comment:
nit: all of the bluesky code imports this as `lexutil` so i've been following that pattern
```suggestion
	lexutil "github.com/bluesky-social/indigo/lex/util"
```

Reply:
rewrote the function elsewhere and it doesn't use lexutil 🫡